project-fortis-pipeline/fortis-deploy.sh

```diff
@@ -50,6 +56,30 @@ throw_if_empty() {
   if [ -z "${value}" ]; then echo "Parameter '${name}' cannot be empty." 1>&2; print_usage; exit -1; fi
 }
 
+throw_if_tls_certificate_info_not_complete() {
+  local hostname="$1"
+  local certificate="$2"
+  local key="$3"
+  if [[ ! -z "$hostname" ]] && [[ ! -z "$certificate" ]] && [[ ! -z "$key" ]]; then
```

Comment:
Nit: can just be single brackets so that we stick to POSIX where possible. Also `! -z` should be equivalent to  `-n`.